.circleci/config.yml

```diff
@@ -24,12 +24,14 @@ jobs:
           docker build "${DOCKER_ARGS}" -t "${DOCKER_ORG}/circleci-fullstack-cypress:${CIRCLE_BRANCH}" -t circleci-fullstack-cypress circleci-fullstack-cypress/
           docker build "${DOCKER_ARGS}" -t "${DOCKER_ORG}/circleci-deploy:${CIRCLE_BRANCH}" -t circleci-deploy circleci-deploy/
           docker build "${DOCKER_ARGS}" -t "${DOCKER_ORG}/circleci-postgres-ram:${CIRCLE_BRANCH}" -t circleci-postgres-ram circleci-postgres-ram/
+          docker build "${DOCKER_ARGS}" -t "${DOCKER_ORG}/dev-stack:${CIRCLE_BRANCH}" -t dev-stack dev-stack/
       - run: |
           docker push "${DOCKER_ORG}/circleci-base:${CIRCLE_BRANCH}"
           docker push "${DOCKER_ORG}/circleci-fullstack:${CIRCLE_BRANCH}"
           docker push "${DOCKER_ORG}/circleci-fullstack-cypress:${CIRCLE_BRANCH}"
           docker push "${DOCKER_ORG}/circleci-deploy:${CIRCLE_BRANCH}"
           docker push "${DOCKER_ORG}/circleci-postgres-ram:${CIRCLE_BRANCH}"
+          docker push "${DOCKER_ORG}/dev-stack:${CIRCLE_BRANCH}"
```

Comment:
This applies to all of them here, but do you think there should be two tags?

One being `latest` and another being a the git hash (or version)

this is so breaking changes are not forced upon everyone reloading the dev-stack (and other images)

Reply:
The way it works is basically the branch name is the tag. This means you use the `master` for what's merged - but you can test what's in a branch simply by using the branch name. That means you don't need to merge to master to test.

Reply:
For sure, it just breaks convention of tagging docker images.

It would also mean if Pollen decided to adopt a different methodology of git practices, e.g. gitflow. Or decided to follow the new approach of using `main` instead of `master` then there would be confusion.

Totally understand though, and best to address this in a seperate ticket as the change is outside the scope of this ticket